### DISCUSSION TITLE
SharePoint (System Module)

### DISCUSSION THREAD

**Discussion by markbisson** [microsoft/ALAppExtensions#23167] [General]
Looking for details on the roadmap for the SharePoint functionality in System Module.  Currently there is no way of updating custom columns with meta-data (list items and files).  We currently have our own solution for SharePoint integration within our vertical solution but would like to phase out and implement System module (if the functionality allows us to).

If anyone could help with a contact or details of roadmap items for this module that would be very much appreciated.  We are happy to contribute also to the module or help in any way we can.

**Comment by pri-kise** [comment]
Currently there is no roadmap for the sharepoint module.
The last improvements were mostly added by partners.

Someone created draft PR for metadata improvements, but this PR wasn't updated for about two months.
https://github.com/microsoft/ALAppExtensions/pull/22546
Maybe you can help on this PR.

**Comment by JesperSchulz** [comment]
Correct. We currently don't really have any public roadmap. The SharePoint module is primarily contribution driven for now. I am hopeful that we soon will take a more coordinated approach to the file handling story all up, but for now our focus is on other things.
If you've got any changes you'd like to get into the product, please don't hesitate to create an issue or PR, where we can discuss your proposed changes further.
